igvm/hypervisor_preferences.py

```diff
@@ -130,6 +130,36 @@ def __call__(self, vm, hv):
         return value is not None and value > self.limit
 
 
+class HypervisorCpuUsageLimit(object):
```

Comment:
Info: The inheritance of object is implicit at least when using Python 3 you don't need to write it.

Reply:
Removed the not needed inheritance.